tools/run_tests/run_interop_tests.py

```diff
@@ -724,6 +726,9 @@ def auth_options(language, test_case, service_account_key_file=None):
     key_file_arg = '--service_account_key_file=%s' % service_account_key_file
     default_account_arg = '--default_service_account=830293263384-compute@developer.gserviceaccount.com'
 
+    # When using google_default_credentials outside of cloud-to-prod, the environment variable
```

Comment:
nit: can we change the comment to have a "TODO" init, just to make clear that the script needs an edit for that to work.